human_trajectory/Trajectory trajectory
bool qsr_visualisation
string[] current_uuids_detected
---
std_msgs/Header header
string uuid
string soma_roi_id
string soma_map
string soma_config
int64 start_time
relational_learner/episodeMsg[] episodes
